module_2/assignment1_2.js

```diff
@@ -0,0 +1,47 @@
+const PI = 3.14;
+let num1 = 3;
+let str = 'a';
+let name = "John";
+let zero = 0;
+let strZero = '0';
+let arr = [];
+let user = {
+  name: 'john',
+  age: 29,
+}
+let nan = NaN;
+let nullValue = null;
+
+console.log("float constant: ", typeof PI);
```

Comment:
Can you add comment as answer in front of all the console.log ?

Reply:
done